ui/components/ui/editable-label/editable-label.js

```diff
@@ -73,12 +81,21 @@ class EditableLabel extends Component {
   }
 
   render() {
-    const { isEditing } = this.state;
-    const { className } = this.props;
+    const { isEditing, value } = this.state;
+    const { className, accountsNames } = this.props;
 
     return (
-      <div className={classnames('editable-label', className)}>
-        {isEditing ? this.renderEditing() : this.renderReadonly()}
+      <div>
```

Comment:
What is the purpose of this wraoper div? If you just want to return two elements here, you can do that using a React fragment rather than creating a new DOM element.

Reply:
Fixed in: d69e6682ef8cce5a6f0d7dd860acabac7ec614e3